tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/FileContentMappingTests.fs

```diff
@@ -1,6 +1,6 @@
 ﻿module TypeChecks.FileContentMappingTests
 
-open NUnit.Framework
+open Xunit
```

Comment:
These tests were not executed at all. They were not rewritten to Xunit whereas the project had xunit adapter set up via dependencies. Now they run.